### DISCUSSION TITLE
Regarding Add New Technician form

### DISCUSSION THREAD

**Discussion by chandula00** [cepdnaclk/e19-co227-digital-reporting-of-technical-visits#79] [Q&A] [answered]
I attempted to style the newly created "Add New Technician Form" and made some minor adjustments. However, I encountered an issue with the labels. When I placed them within a flex container, the width of each input field changed based on the label's length. I couldn't pinpoint the exact reason behind this behaviour. Could you help me identify the cause?

Additionally, the pop-up card appears to be slightly oversized. Could you adjust its size? I believe it would be more efficient for you to make this adjustment since the entire JSX code and SCSS were initially created by you. As a temporary solution, I modified the scale of the card. 😶‍🌫️

**Comment by Bimbara28** [comment, marked as answer]
Thank you for your response. It appears that the issue with the labels' behavior may have been due to the styling of the outer div as a flex container without properly styling the inner div. I appreciate you noting this, and I'm confident that addressing this styling oversight will resolve the problem.😶‍🌫️

Regarding the size of the popup card, I understand your point about the component's placement and the change in the parent element. Since the popup card's dimensions were specified in "em," it makes sense that they might have been affected by this change, as "em" dimensions are relative to the parent element. I'm looking forward to the adjustments you'll make to ensure the correct sizing of the popup card.💫

**Comment by G5andeepD** [comment]
I Added A Technician Form for editing as well. Hope it is convienient

**Reply by chandula00** [reply]
Cupiri 🔥
